### ISSUE TITLE
[false-positive] CVE-2017-5487

### ISSUE THREAD

**Issue by 0xcrypto** [projectdiscovery/nuclei-templates#1934]
**Nuclei template**:  [cves/2017/CVE-2017-5487.yaml](https://github.com/projectdiscovery/nuclei-templates/blob/master/cves/2017/CVE-2017-5487.yaml) 

This template is too generic. All WordPress target with REST API enabled will result positive.

**Comment by gy741** [comment]
It would be great if you could provide a test site.
Vulnerable sites and sites that generate false positives.

**Comment by 0xcrypto** [comment]
>  Vulnerable sites and sites that generate false positives.
https://elementor.com/?rest_route=/wp/v2/users/

The severity also should be `info` not  low` as there is little to no security risk in disclosing usernames and avatar URLs.

**Comment by ehsandeep** [comment]
@gy741 title is not accurate, looks like this issue is about severity.

@0xcrypto Yes it's generic for `WordPress` sites, we kept it as it has known CVE, for future consider using `exclude-template` flag or config file to remove anything specific you don't want to see in your results.

We have updated this severity to info, as this is directly not an issue https://github.com/projectdiscovery/nuclei-templates/commit/3bbcb23cd083a57d56f2219316d0d51d8b4c5046 but for any other template, template exclusion is the approach to follow.
